subsys/mgmt/ec_host_cmd/ec_host_cmd_handler.c

```diff
@@ -196,15 +271,15 @@ FUNC_NORETURN static void ec_host_cmd_thread(void *hc_handle, void *arg2, void *
 		/* Wait until RX messages is received on host interface */
 		k_sem_take(&rx->handler_owns, K_FOREVER);
 
+		ec_host_cmd_log_request(rx->buf);
 		status = verify_rx(rx);
 		if (status != EC_HOST_CMD_SUCCESS) {
-			send_status_response(hc->backend, tx, status);
+			ec_host_cmd_send_response(status, &args);
```

Comment:
unrelated change?

Reply:
It is related. I want to use one function to send all responses. It allows adding logging only to that function. I've created a separated commit for it.